release_dashboard/lib/services/local_conductor.dart

```diff
@@ -60,6 +61,16 @@ class LocalConductorService extends ConductorService {
     return null;
   }
 
+  Checkouts get checkouts {
```

Comment:
this will create a new `Checkouts` object each time it's called. I think we should probably do:
```suggestion
  late final Checkouts checkouts = Checkouts(
      parentDirectory: rootDirectory,
      processManager: processManager,
      fileSystem: fs,
      platform: platform,
      stdio: stdio,
    );
```

And then have the object created on line 85 also reference this same one.

Reply:
I added a change on line 85, but that didn't get committed for some reason.